lms/services/h_api.py

```diff
@@ -33,14 +33,10 @@ def get_user(self, username):
 
         :rtype: HUser
         """
-        userid = HUser(self._authority, username).userid
+        userid = HUser(username).userid(self._authority)
```

Comment:
`HAPIService` was also already using the authority. This is the other "edge" of the app, where we have to render userid's and groupid's containing the authority for sending to the h API

Reply:
I'm guessing this is out of sequence with the other PRs, as this has all shipped out!